todo_app/view_model.py

```diff
@@ -0,0 +1,30 @@
+from todo_app.data.trello_items import Item
+
+class ViewModel:  
+    def __init__(self, retrieve_todo: list[Item], retrieve_doing: list[Item], retrieve_done: list[Item]):
+        combined_lists = retrieve_todo + retrieve_doing + retrieve_done
+        self._items = combined_lists
+ 
+    @property
+    def retrieve_todo(self) -> list[Item]:
+        results: list[Item] = []   
+        for each_item in self._items:
+            if each_item.status == "To Do":
+                results.append(each_item)
+        return results
```

Comment:
This is nice and clean! I like to offer a comparison with the one-liner list comprehension below; this is much more concise but it's not obvious to me whether it's more readable/maintainable.

```python
results = [each_item for each_item in self._items if each_item.status == "To Do"]
return results
```